manager/manager.py

```diff
@@ -415,11 +415,18 @@ def insert_job(self, job):
         self.job_db_handler.insert_item(job_data)
 
     def query_jobs(self, data=None):
-        items = self.job_db_handler.query_items(data)
+        items, last_key = self.job_db_handler.query_items(data)
         jobs = []
         if items and len(items):
             for item in items:
                 jobs.append(TxJob(item))
+
+        while last_key != None:
```

Comment:
I guess it is in this PR I should have been talking about moving the last_key recursive call into query_items() so that anything that calls query_items() gets all the items, unless they specify a limit. I see both PRs have this code...will it conflict?